### PR TITLE
Assignment3 @PlantBrush

### PR DESCRIPTION
**Pull request by NazariyPogo**
URL: https://lemon-ground-09e5fcf1e.1.azurestaticapps.net

### CODE REVIEW

**Comment by A-Tanner** [comment]
I will review this

File: wordle-web/pages/game.vue [hunk 1]
```diff
@@ -52,7 +72,15 @@
...
-          <v-btn class="ml-2" @click="resetGame"> Play Again? </v-btn>
+          <v-btn class="ml-2" color="primary" @click="resetGame">
+            Play Again?
+          </v-btn>
+          <v-btn color="primary" :disabled="saved" @click="checkForName()">
+            Save Score?
+          </v-btn>
+          <v-btn color="primary" :disabled="unsure" @click="areYouSure()">
+            Are you sure?
+          </v-btn>
```
Comment (A-Tanner) [inline wordle-web/pages/game.vue:83]: This appears to display behind the win alert, making it very difficult to see

File: Wordle.Api.Tests/PlayerServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+        [TestMethod]
+        public void GetPlayers()
+        {
+            PlayerService sut = new(_context);
+
+            Assert.AreEqual(1, sut.GetTenPlayers().Count());
+        }
+    }
+}
```
Comment (A-Tanner) [inline Wordle.Api.Tests/PlayerServiceTests.cs:32]: The core features of PlayerService have not been tested

**Review by A-Tanner — COMMENTED** [review]
Really solid work folks!

File: wordle-web/pages/game.vue (hunk 1 above)
Comment (NazariyPogo) [reply wordle-web/pages/game.vue:83]: The 'Are you sure?' is meant as a confirmation for if the person has the name 'Guest'. If both buttons are disabled, than the game has been saved. If you choose a name before saving, the second button does not appear

**Comment by linhtran8699** [comment]
After i am done playing the game, and want to save the score, I clicked on the "save score" button, the button only got dimmed. The guest name button is just a gray box without any letter on it, but after I click that box and save a name, the box will display the saved name.

File: wordle-web/pages/game.vue [hunk 3]
```diff
@@ -11,7 +11,27 @@
...
 
-    <v-container v-if="isLoaded">
+  <v-container v-if="isLoaded">
+      <v-row justify="end">
+        <v-btn>
+          <v-container @click="toggleDialog">
+            {{ newName }}
+          </v-container>
+
+          <v-dialog v-model="dialog" width="450" persistent>
```
Comment (mmwoodfo) [inline wordle-web/pages/game.vue:21]: To get the dialog to appear automatically at the end of the game, do something like this in your typescript:
```suggestion
          @Watch("this.wordleGame.state")
           function(){
                 if (this.wordleGame.state === GameState.Won) {
                   // something to check if the player name is empty or guest
                   this.dialog=true
                  }
           }
```

**Pull request closed without merging**